.circleci/config.yml

```diff
@@ -82,13 +82,32 @@ commands:
       - run:
           name: Build tests
           command: |
-            cargo test --no-run --all
+            cargo test --no-run --workspace --exclude crml-cennzx-spot --exclude crml-sylo
   cargo-run-test:
     steps:
       - run:
           name: Run tests
           command: |
-            cargo test --all
+            cargo test --workspace --exclude crml-cennzx-spot --exclude crml-sylo
+  cargo-test-coverage:
+    steps:
+      - run:
+          name: Report test coverage
+          command: |
```

Comment:
Not a big deal, but thinking if any pros/cons to put multiple commands into a shell script file like `scripts/test_coverage.sh`

Reply:
Good idea

Reply:
Done